xgcm/test/test_faceconnections.py

```diff
@@ -126,38 +126,6 @@ def cubed_sphere_connections():
     }
 
 
-def test_create_periodic_grid(ds):
-    ds = ds.isel(face=0)
-    grid = Grid(ds, periodic=True)
-    for axis_name in grid.axes:
-        axis = grid.axes[axis_name]
-        assert axis._facedim is None
-        connect_left, connect_right = axis._connections[None]
-        assert connect_left[0] is None
-        assert connect_right[0] is None
-        assert connect_left[1] is axis
-        assert connect_right[1] is axis
-        assert connect_left[2] is False
-        assert connect_right[2] is False
-
-
-def test_get_periodic_grid_edge(ds):
-    ds = ds.isel(face=0)
-    grid = Grid(ds, periodic=True)
-
-    xaxis = grid.axes["X"]
-    left_edge_data_x = xaxis._get_edge_data(ds.data_c)
-    np.testing.assert_allclose(left_edge_data_x, ds.data_c.isel(x=-1).data[:, None])
-    right_edge_data_x = xaxis._get_edge_data(ds.data_c, is_left_edge=False)
-    np.testing.assert_allclose(right_edge_data_x, ds.data_c.isel(x=0).data[:, None])
-
-    yaxis = grid.axes["Y"]
-    left_edge_data_y = yaxis._get_edge_data(ds.data_c)
-    np.testing.assert_allclose(left_edge_data_y, ds.data_c.isel(y=-1).data[None, :])
-    right_edge_data_y = yaxis._get_edge_data(ds.data_c, is_left_edge=False)
-    np.testing.assert_allclose(right_edge_data_y, ds.data_c.isel(y=0).data[None, :])
-
-
 def test_connection_errors(ds):
```

Comment:
Not sure we should leave this just as a passing test. Seems like this is an older test. Mayne @rabernat remembers what this was supposed to test? Probably something with proper error handling when supplying face connections?

Reply:
This question seems unconnected to this specific PR